gcs/pom.xml

```diff
@@ -220,9 +223,12 @@
                   <include>com.google.oauth-client</include>
                   <include>com.google.protobuf</include>
                   <include>com.google.storage.v2</include>
+                  <include>com.lmax</include>
                   <include>io.grpc</include>
                   <include>io.opencensus</include>
                   <include>io.perfmark</include>
+                  <include>org.apache.httpcomponents</include>
+                  <include>org.threeten:threetenbp</include>
```

Comment:
Does not look like this is something that we need: https://github.com/ThreeTen/threetenbp